index.html

```diff
@@ -16156,6 +16156,7 @@ <h2>Including Elements in the Accessibility Tree</h2>
             <ul>
               <li>Elements that are currently focused, even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.</li>
               <li>Elements that are a valid target of an <pref>aria-activedescendant</pref> attribute.</li>
+              <li>Elements that have <a>live region</a> roles or attributes.</li>
```

Comment:
This addition is editorial, since the section already specifies elements that can fire an accessibility API event, but I thought it should be added explicitly since it was the original example that kicked this off (and is probably the other most common attribute authors use that would trigger an a11y event)